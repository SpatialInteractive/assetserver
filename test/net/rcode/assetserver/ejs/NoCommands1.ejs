This is a file
with no special directives.
It should produce one Literal with a start line of 1 and an end line of 3.