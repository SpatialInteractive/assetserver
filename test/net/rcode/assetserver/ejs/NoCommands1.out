{Literal(1,0,117)='This is a file\nwith no special directives.\nIt should produce one Literal with a start line of 1 and an end line of 3.'}